server/scripts/redirector.py

```diff
@@ -144,13 +144,16 @@ def resolve_doc_path(self, channel):
 
   def get_channel(self):
     """Quick accessor to examine a request and determine what channel
-    (dev/stable/etc) we're looking at. Return None if we have a weird unexpected
-    URL."""
-    for channel in ApiDocs.latest_versions.keys():
-      if self.request.path.startswith('/apidocs/channels/%s' % channel):
-        return channel
-      if self.request.path.startswith('/%s' % channel):
-        return channel
+    (be/beta/dev/stable) we're looking at. Return None if we have a weird
+    unexpected URL."""
+    if self.request.path.startswith('/apidocs/channels/'):
+      channel = self.request.path[18:]
+    elif self.request.path.startswith('/'):
```

Comment:
Could we split first, and then just drop "apidocs"and "channels", or "".  That seems cleaner than counting characters.  Are there really cases where channel doesn't begin with "/"?  We don't need to handle that case, I think.

Reply:
Done, I found no documentation saying that request.path always starts with the slash and the old code checked it, so I kept that just in case.